src/PowerShellEditorServices.Host/EditorServicesHost.cs

```diff
@@ -243,7 +246,7 @@ public void StartLogging(string logFilePath, LogLevel logLevel)
             foreach (string module in this.additionalModules)
             {
                 await this.editorSession.PowerShellContext.ExecuteCommand<System.Management.Automation.PSObject>(
-                    new System.Management.Automation.PSCommand().AddCommand("Import-Module").AddArgument(module),
+                    new System.Management.Automation.PSCommand().AddCommand("Microsoft.PowerShell.Core\\Import-Module").AddArgument(module),
                     false,

```

Comment:
Maybe a good time to label these arguments too?

Reply:
You mean `.AddCommand("Microsoft.PowerShell.Core\\Import-Module").AddParameter("Name", module),`?  If so, yeah, makes senses to me.

Reply:
Oh, I meant the `true` and `false` in C#. But I'm in favour of all of it :)